dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/IChatCompletion.cs

```diff
@@ -16,7 +16,7 @@ public interface IChatCompletion
     /// <returns>Generated chat message in string format</returns>
     public Task<string> GenerateMessageAsync(
         ChatHistory chat,
-        ChatRequestSettings requestSettings,
+        ChatRequestSettings? requestSettings = null,
```

Comment:
Do we expect all chat-style models to have the same (or very similar) properties as OpenAI? i.e. Is this specific settings object required in the IChatCompletion interface, or does that effectively make it an IOpenAIChatCompletion?

Reply:
we'll revisit the interface once we have more chat models. We have a similar need with image generation taking different params